### ISSUE TITLE
EstimateGasBundle returns {"error":"unknown method: eth_estimateGasBundle"}

### ISSUE THREAD

**Issue by apecollector** [cryptoriums/flashbot#5]
Trying to estimate the gas for a bundle of transactions and getting a response status 400 with error:

```
{"error":"unknown method: eth_estimateGasBundle"} reqDump:POST / HTTP/1.1
Host: relay.flashbots.net
```

**Comment by krasi-georgiev** [comment]
I have no idea, you need to look if flashbots have changed their API, PR is welcome for a fix.
I haven't used this endpoint in a while.

**Comment by apecollector** [comment]
@krasi-georgiev ok thanks, my use case is in preparing a bundle where some of the transactions are only valid after the first ones. Basically trying to rescue erc20 / nfts from hacked wallets that are immediately drained of any deposited eth. The typical tx bundle looks like:

1. Send enough ETH to cover gas for the remaining tx's (this is why I need eth_estimateGasBundle)
2. Unstake erc20 tokens
3. Unstake nfts out
4. Transfer erc20 tokens
5. Transfer nfts out 

So basically tx 4 and 5 cannot estimate gas with normal estimate gas means so I cannot estimate how much gas to send in tx 1. If I send more than enough the hacked wallet gets drained and we lose any remainder.

**Comment by apecollector** [comment]
I also found this related issue: https://github.com/flashbots/mev-relay-js/issues/67

**Comment by krasi-georgiev** [comment]
aah yeah, now I remember, this method is present in geth, but not exposed in the relay.
you can use callBundle to estimate the gas.

**Comment by apecollector** [comment]
Thanks, I'll try that out, just starting with flashbots!
